data/MacOSX/Makefile.am

```diff
@@ -48,22 +49,30 @@ install-exec-hook:
 		$(srcdir)/run-ssr.applescript
 
 install-data-hook:
-	cp -rf $(QTLIBDIR)/QtGui.framework/Resources/qt_menu.nib $(DESTDIR)$(pkgdatadir)/
+	mkdir -p $(DESTDIR)$(contentsdir)/PlugIns/platforms
+	cp -f `qmake -query QT_INSTALL_PLUGINS`/platforms/libqcocoa.dylib $(DESTDIR)$(contentsdir)/PlugIns/platforms
+	install_name_tool -id @executable_path/../PlugIns/platforms/libqcocoa.dylib $(DESTDIR)$(contentsdir)/PlugIns/platforms/libqcocoa.dylib
+
 	-cd $(DESTDIR)$(dmgrootdir) && $(LN_S) /Applications
-	for qtc in QtGui QtCore QtOpenGL; do \
-		mkdir -p $(prefix)/Frameworks/$$qtc.framework/Versions/4 \
-		; cp $(QTLIBDIR)/$$qtc.framework/Versions/4/$$qtc $(prefix)/Frameworks/$$qtc.framework/Versions/4/ \
-		; chmod +w $(prefix)/Frameworks/$$qtc.framework/Versions/4/$$qtc \
-		; install_name_tool -id @executable_path/../Frameworks/Versions/4/$$qtc.framework/$$qtc $(prefix)/Frameworks/$$qtc.framework/Versions/4/$$qtc \
+	for qtc in QtGui QtCore QtOpenGL QtWidgets QtPrintSupport; do \
+		install_name_tool -change $(QTLIBDIR)/$$qtc.framework/Versions/5/$$qtc @executable_path/../Frameworks/$$qtc.framework/Versions/5/$$qtc $(DESTDIR)$(contentsdir)/PlugIns/platforms/libqcocoa.dylib \
+		; mkdir -p $(prefix)/Frameworks/$$qtc.framework/Versions/5 \
+		; cp $(QTLIBDIR)/$$qtc.framework/Versions/5/$$qtc $(prefix)/Frameworks/$$qtc.framework/Versions/5/ \
+		; chmod +w $(prefix)/Frameworks/$$qtc.framework/Versions/5/$$qtc \
+		; install_name_tool -id @executable_path/../Frameworks/Versions/5/$$qtc.framework/$$qtc $(prefix)/Frameworks/$$qtc.framework/Versions/5/$$qtc \
+		; install_name_tool -change $(QTLIBDIR)/QtCore.framework/Versions/5/QtCore @executable_path/../Frameworks/QtCore.framework/Versions/5/QtCore $(prefix)/Frameworks/$$qtc.framework/Versions/5/$$qtc \
+		; install_name_tool -change $(QTLIBDIR)/QtGui.framework/Versions/5/QtGui @executable_path/../Frameworks/QtGui.framework/Versions/5/QtGui $(prefix)/Frameworks/$$qtc.framework/Versions/5/$$qtc \
+		; install_name_tool -change $(QTLIBDIR)/QtWidgets.framework/Versions/5/QtWidgets @executable_path/../Frameworks/QtWidgets.framework/Versions/5/QtWidgets $(prefix)/Frameworks/$$qtc.framework/Versions/5/$$qtc \
+
```

Comment:
Noticed this too when building on linux and weirdly enough, the empty line is not there on OSX. Let's see what happens when removing it on linux